scripts/cnn_variant_wdl/cram2filtered.wdl

```diff
@@ -0,0 +1,473 @@
+  }
+}
+
+
+task RunHC4 {
+    File input_bam
+    File input_bam_index
+    File reference_fasta
+    File reference_dict
+    File reference_fasta_index
+    String output_prefix
+    File interval_list
+    String extra_args
+    File? gatk_override
+
+    # Runtime parameters
+    Int? mem_gb
+    String gatk_docker
+    Int? preemptible_attempts
+    Int? disk_space_gb
+    Int? cpu 
+    Boolean use_ssd = true
+
+    # You may have to change the following two parameter values depending on the task requirements
+    Int default_ram_mb = 8000
+    # WARNING: In the workflow, you should calculate the disk space as an input to this task (disk_space_gb).
+    Int default_disk_space_gb = 200
+
+    # Mem is in units of GB but our command and memory runtime values are in MB
+    Int machine_mem = if defined(mem_gb) then mem_gb *1000 else default_ram_mb
+    Int command_mem = machine_mem - 1000
+
+    command {
+        set -e
+        export GATK_LOCAL_JAR=${default="/root/gatk.jar" gatk_override}
+
+        #gatk --java-options "-Xmx${command_mem}m" \ 
+        java "-Xmx${command_mem}m" -jar ${gatk_override} \
+        HaplotypeCaller \
+        -R ${reference_fasta} \
+        -I ${input_bam} \
+        -O ${output_prefix}_hc4.vcf.gz \
+        -L ${interval_list} \
+        -bamout ${output_prefix}_bamout.bam \
+        ${extra_args}
+    }
+
+    output {
+        File bamout = "${output_prefix}_bamout.bam"
+        File bamout_index = "${output_prefix}_bamout.bai"
+        File raw_vcf = "${output_prefix}_hc4.vcf.gz"
+        File raw_vcf_index = "${output_prefix}_hc4.vcf.gz.tbi"
+    }
+    runtime {
+        docker: gatk_docker
+        memory: machine_mem + " MB"
+        # Note that the space before SSD and HDD should be included.
+        disks: "local-disk " + select_first([disk_space_gb, default_disk_space_gb]) + if use_ssd then " SSD" else " HDD"
+        preemptible: select_first([preemptible_attempts, 3])
+        cpu: select_first([cpu, 1])  
+    }
+}
+
+task CNNScoreVariants {
+    String input_vcf
+    File input_vcf_index
+    File reference_fasta
+    File reference_dict
+    File reference_fasta_index
+    String? bam_file
+    String? bam_file_index
+    File architecture_json
+    File architecture_hd5
+    String tensor_type
+    String output_prefix
+    Int inference_batch_size
+    Int transfer_batch_size
+    File interval_list
+    File? gatk_override
+
+    # Runtime parameters
+    Int? mem_gb
+    String gatk_docker
+    Int? preemptible_attempts
+    Int? disk_space_gb
+    Int? cpu 
+    Boolean use_ssd = true
+
+    String bam_cl = if defined(bam_file) then "-I ${bam_file}" else " "
+
+    # You may have to change the following two parameter values depending on the task requirements
+    Int default_ram_mb = 16000
+    # WARNING: In the workflow, you should calculate the disk space as an input to this task (disk_space_gb).  Please see [TODO: Link from Jose] for examples.
```

Comment:
Again, this should be pretty consistent -- especially for single-sample VCFs, which is what this pipeline is for.  You can hard code the disk space at 200.